omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/ContinuityOfCareFragmentController.java

```
@@ -34,8 +34,12 @@ public void viewCCD() {
         LOGGER.info("View CCD");
     }
 
-    public void importCCD(@RequestParam("patientId") Integer patientId, OutputStream response) {
-        CcdService service = Context.getRegisteredComponent("ccdService", CcdService.class);
-        service.downloadCcdAsPDF(response, Context.getPatientService().getPatient(patientId));
+    public void importCCD(@RequestParam("patientId") Integer patientId, HttpServletResponse response) {
+        try {
+            CcdService service = Context.getRegisteredComponent("ccdService", CcdService.class);
+            service.downloadCcdAsPDF(response.getOutputStream(), Context.getPatientService().getPatient(patientId));
+        } catch (Exception ex) {
+            LOGGER.error(ex.getMessage(), ex);
```

lets just rethrow the exception